corehq/apps/geospatial/static/geospatial/js/geo_config.js

```diff
@@ -51,8 +52,26 @@ hqDefine("geospatial/js/geo_config", [
         });
 
         self.selectedAlgorithm = ko.observable();
+        self.minCasesPerUser = ko.observable(data.min_cases_per_user);
+        self.maxCasesPerUser = ko.observable(data.max_cases_per_user);
+
+        if (!data.max_cases_per_user) {
+            $("#div_id_max_cases_per_user").hide();
+        }
+
         self.plaintext_api_token = ko.observable(data.plaintext_api_token);
 
+        self.onSpecifyMaxCasesPerUser = function () {
+            var $maxCasesPerUserInput = $("#div_id_max_cases_per_user");
+            var $maxCasesPerUserCheckbox = $("#max_cases_per_user_checkbox_id");
```

Comment:
nit: Is it a practice to use "_id" in the ID of HTML elements? We added div_id in the beginning. Good to check once about what is followed in HQ.